### ISSUE TITLE
mac os vgg 训练内存耗尽问题

### ISSUE THREAD

**Issue by albertyou2** [dmlc/mxnet#2631]
首先抱歉用中文，英文我担心描述不清楚。这是我第一次在github上提issue，如果有什么不合理的地方请包涵。毕竟觉得mxnet非常好用，所以希望它越来越好！

我使用的是mxnet 0.7.0，图像的格式是jpg，只有400张左右.
0.环境配置：我在mac os 10.11.4,没有使用cuda,使用了kv，使用了openmp，openblas(按照官方文档mac os下的build方法编译的)

1.我的数据格式是，shape=(3,200,200),batch_size=10
2.我直接使用的例子中的vgg模型，分类改成了3类
3.训练使用的就是train_model.py，命令为
python train_albertnet.py --data-dir=/Users/zuekiWork/Documents/mxnet_code --lr=0.0005 --lr-factor=0.994 --lr-factor-epoch=5
4.feed数据使用的是
 train = mx.io.ImageRecordIter(
        path_imgrec = os.path.join(args.data_dir, args.train_dataset),
        shuffle     = True,
        data_shape  = data_shape,
        batch_size  = args.batch_size,
        rand_crop   = True,
        rand_mirror = True,
        num_parts   = kv.num_workers,
        part_index  = kv.rank)

    val = mx.io.ImageRecordIter(
        path_imgrec = os.path.join(args.data_dir, args.val_dataset),
        shuffle     = True,
        # mean_img    = args.data_dir + "val.bin",
        rand_crop   = True,
        rand_mirror = True,
        data_shape  = data_shape,
        batch_size  = args.batch_size,
        num_parts   = kv.num_workers,
        part_index  = kv.rank)
现象：.当我用shape(3,32,32)可以正常工作,内存大小在2g左右；当我改变成(3,200,200),内存会涨到63G左右然后被系统干掉

是我哪里使用错误了么？还是说需要限制内存的使用？

**Comment by albertyou2** [comment]
我想我找到原因了，因为我把vgg 中的 pool3,pool4,pool5 的 stride(2,2)删除了导致的。非常抱歉
